tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,64 @@
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+
+class TestContactsRTL(GaiaImageCompareTestCase):
+
+    def setUp(self):
+       GaiaImageCompareTestCase.setUp(self)
+
+       self.push_resource('IMG_0001.jpg')
+
+    def test_contacts_app(self):
+
+        contacts = Contacts(self.marionette)
+        contacts.launch()
+        self.take_screenshot('contacts')
+        view_settings = contacts.tap_settings()
+        self.take_screenshot('contacts-setting')
+        view_settings.tap_import_contacts()
+        self.take_screenshot('contacts-import')
+        view_settings.tap_import_from_sim()
+        self.take_screenshot('contacts-sim_import')
```

Comment:
oh, you don't need to use the same test account that the test harness uses,  you can put your personal email info on testvars, and when it's submitted, it'll use the test account defined in the jenkins testvars.  To the script, this will make no difference because it gets the information from the same testvar location.  What you need to know is that the syntax of the testvar section where it stores the email information.